### ISSUE TITLE
Automation to add all tfp distributions to genjax.

### ISSUE THREAD

**Issue by georgematheos** [genjax-community/genjax#1459]
Occasionally I run into the fact that one of the distributions from tensorflow probability does not exist in genjax.  (I just ran into this with the `gamma` distribution.).  A suggestion for a potential solution that would fully solve this issue, rather than having us solve it by adding one-off distributions: add a script that automatically populates GenJAX with the correct wrapper for every probability distribution defined under `tfp.distributions`.

**Comment by sritchie** [comment]
Good idea. I also want to modify these to pass through keyword args etc so we can use those too… distributions are on my hit list for tidying up, keep the ideas coming

**Comment by sritchie** [comment]
Closed via https://linear.app/chisym/issue/GEN-901, see the notes there!
